test/IR/load-store.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: mlir-opt %s -split-input-file | FileCheck %s
+
```

Comment:
These tests can do into test/IR/core-ops.mlir to live together with other load/store tests